pkg/operation/common/utils.go

```diff
@@ -334,7 +334,7 @@ func DeleteGrafanaByRole(ctx context.Context, k8sClient kubernetes.Interface, na
 		return err
 	}
 
-	if err := k8sClient.Client().DeleteAllOf(ctx, &extensionsv1beta1.Ingress{}, deleteOptions...); client.IgnoreNotFound(err) != nil {
+	if err := k8sClient.Client().DeleteAllOf(ctx, &networkingv1.Ingress{}, deleteOptions...); client.IgnoreNotFound(err) != nil {
```

Comment:
Will this work for K8s version <1.21 too? Did you test it?
https://github.com/gardener/gardener/blob/697624eda679a351ac988803da666ecaa82f1232/pkg/operation/seed/nginxingress.go#L185-L186

Reply:
I tested for 1.21 not below.

Reply:
Then you might want to test it for < 1.21 case too.

Reply:
I tested it for 1.19.16 this works fine.

Reply:
Okay, Thanks!